scripts/prepare_gnomAD.py

```diff
@@ -0,0 +1,231 @@
+"""Subset and annotate gnomAD hail table"""
+
+import gnomad.utils.vep
+import hail as hl
+from typing import Union
+
+
+def get_worst_consequence_with_non_coding(ht):
+    """
+    copied and adapted from gnomad_lof
```

Comment:
Would be nice to add a lint to the original to get some context. 
The function is hard to read, so if you could add docstrings and comments, and clarify which parts have you changed from the gnomad_lof codebase, that would help a lot to review it!